packages/contracts-bedrock/src/Safe/GetSigners.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
Perhaps the library code should be `^0.8.0` like our other libraries? Makes it more portable